package.json

```diff
@@ -110,15 +110,20 @@
     "handlebars": "^4.7.8",
     "ical-generator": "^7.1.0",
     "markdown-it": "^14.1.0",
-    "mongoose": "^8.4.0",
+    "mongoose": "^8.4.3",
     "ngx-cookie-service-ssr": "^17.1.0",
     "ngx-countup": "^13.1.0",
     "reflect-metadata": "^0.1.14",
     "rxjs": "~7.8.1",
-    "tslib": "^2.6.2",
+    "tslib": "^2.6.3",
     "uuid": "^9.0.1",
     "web-push": "^3.6.7",
     "zone.js": "0.14.3"
   },
-  "resolutions": {}
+  "resolutions": {},
+  "pnpm": {
+    "overrides": {
```

Comment:
Instead of pnpm.overrides it should be placed in resolutions because that is more compatible.